docs/sources/dashboards/dashboard-public.md

```diff
@@ -0,0 +1,48 @@
+---
+aliases:
+  - /docs/grafana/latest/dashboards/dashboard-manage/
+  - /docs/grafana/latest/features/dashboard/dashboards/
+title: Public dashboards
+weight: 8
+---
+
+## Public dashboards
+
+> **Note:** This alpha-based feature is opt-in.
```

Comment:
Alpha-based seems.. a little funny. How about: "This is an opt-in alpha feature"